### ISSUE TITLE
Could have a second parameter on listener to assess what key changed on store

### ISSUE THREAD

**Issue by ianpierreg** [Spyna/react-context-hook#9]
Could have a second parameter on listener to assess what key changed on store. It would be very helpful!

**Comment by Spyna** [comment]
Sure, 
great feature request, I'll ship it asap.

Thank you

**Comment by Spyna** [comment]
The feature is added, I will release it asap. 
The *listener* function now takes 4 arguments, as you can see [in the doc](https://github.com/Spyna/react-context-hook#advanced-use-of-withstore)

```JS
listener: (state, key, prevValue, nextValue) => {
      console.log(`the key "${key}" changed in the store`)
      console.log('the old value is', prevValue)
      console.log('the current value is', nextValue)
      console.log('the state is', state)
  },
```

**Comment by Spyna** [comment]
released version  v2.1.0 that contains this feature.
